src/Api/RequestAttributesExtractor.php

```diff
@@ -29,19 +31,17 @@
      *
      * @throws RuntimeException
      *
-     * @return array
+     * @return array [$resourceClass, $collectionOperation, $itemOperation, $format]
```

Comment:
Hence the VO I wanted to introduce at some point :/ But what's important is that the user doesn't have to do a step by step debugging or a dump to understand what values are returned... the doc should be informative enough to avoid that